src/source/Crypto/Dtls.h

```diff
@@ -11,14 +11,17 @@
 extern "C" {
 #endif
 
-#define MAX_SRTP_MASTER_KEY_LEN 16
-#define MAX_SRTP_SALT_KEY_LEN 14
-
-#define GENERATED_CERTIFICATE_BITS 2048
-#define GENERATED_CERTIFICATE_SERIAL 0
-#define GENERATED_CERTIFICATE_DAYS 365
-#define GENERATED_CERTIFICATE_NAME (PUINT8) "KVS-WebRTC-Client"
-#define KEYING_EXTRACTOR_LABEL "EXTRACTOR-dtls_srtp"
+#define MAX_SRTP_MASTER_KEY_LEN             16
+#define MAX_SRTP_SALT_KEY_LEN               14
+#define MAX_DTLS_RANDOM_BYTES_LEN           32
+#define MAX_DTLS_MASTER_KEY_LEN             48
+
+#define GENERATED_CERTIFICATE_MAX_SIZE      4096
+#define GENERATED_CERTIFICATE_BITS          2048
+#define GENERATED_CERTIFICATE_SERIAL        1
```

Comment:
Not sure why this change though.

Reply:
This is the problem that I mentioned a while ago that mbedtls has a bug, when it's zero, mbedtls will not append a bit of 0, https://github.com/ARMmbed/mbedtls/issues/2166.